create-aleo-app/template-react-typescript/.eslintrc.cjs

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
would maybe configure this a little more for typescript but fine for now we can iterate